pkg/test/compute/data_locality_bid_test.go

```diff
@@ -0,0 +1,34 @@
+//go:build integration || !unit
+
+package compute
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type DataLocalityBidSuite struct {
+	AskForBidSuite
+}
+
+func TestDataLocalityBidSuite(t *testing.T) {
+	suite.Run(t, new(AskForBidSuite))
+}
```

Comment:
@simonwo ran into some issues here while working on https://github.com/bacalhau-project/bacalhau/pull/2712 Is this logic right? It looks like we never instantiate the `DataLocalityBidSuite` meaning its tests don't get run?